lib/dogtrainer/api.rb

```diff
@@ -303,7 +320,7 @@ def upsert_monitor(
 
     # Create a monitor that doesn't already exist; return its id
     #
-    # @param mon_name [String] mane of the monitor to create
+    # @param _mon_name [String] mane of the monitor to create
```

Comment:
This documented params doesn't seem to be used anywhere. Was it deprecated at some point?